docs/guides/nrfconnect_android_commissioning.md

```diff
@@ -29,240 +26,63 @@ onto a CHIP-enabled Thread network. The instructions are also valid for
 
 The commissioning process is composed of the following main stages:
 
--   CHIPTool discovers a CHIP accessory device over Bluetooth LE.
--   CHIPTool establishes a secure channel to the device over Bluetooth LE, and
+1.  CHIPTool discovers a CHIP accessory device over Bluetooth LE.
+2.  CHIPTool establishes a secure channel to the device over Bluetooth LE, and
     sends CHIP operational credentials and Thread provisioning data.
--   The accessory device joins a CHIP-enabled Thread network.
+3.  The accessory device joins a CHIP-enabled Thread network.
 
 Bluetooth LE is only used during the commissioning phase. Afterwards, only the
-IP connectivity between the smartphone and the accessory device is needed to
-send operational messages. Since a typical smartphone does not have a Thread
-radio built-in, extra effort is needed to prepare a fully-fledged testing
-environment. This page describes how to build a Thread Border Router using a PC
-with a spare Wi-Fi card and an
-[OpenThread Radio Co-Processor](https://openthread.io/platforms/co-processor)
-device.
+IPv6 connectivity between the smartphone and the accessory device is needed to
+send operational messages. The IPv6 address of the device is not exchanged
+during the commissioning process and CHIPTool must use DNS Service Discovery
+(DNS-SD) to learn or refresh the address before the IPv6-based communication can
+be initiated.
+
+Since a typical smartphone does not have a Thread radio built-in, extra effort
+is needed to prepare a fully-fledged testing environment. This page refers to
+the [OpenThread Border Router](openthread_border_router_pi.md) article to set up
+a Thread Border Router on a Raspberry Pi.
```

Comment:
```suggestion
Since a typical smartphone does not have a Thread radio, you need to prepare a fully-fledged testing environment. Refer to
the [OpenThread Border Router](openthread_border_router_pi.md) article to set up
a Thread Border Router on a Raspberry Pi.
```